### PR TITLE
I've removed some debug stuff & fix php 5.3 incompatibility

### PR DESCRIPTION
**Pull request by hempalex**

### CODE REVIEW

File: src/Stnvh/Partial/Zip.php [hunk 1]
```diff
@@ -64,9 +64,6 @@ private function receiveData($ch, $data) {
 	 * @return void
 	 */
 	public function __construct($url, $file = false) {
-		set_time_limit(15);
-		ob_start();
-
```
Comment (stnvh) [inline src/Stnvh/Partial/Zip.php:69]: This is deliberate; I use `ob_start()` in this instance to start capturing any unnecessary output, to then `ob_flush()` before setting headers to output the decompressed file contents. Saves any issues when setting headers after contents is output.

EDIT:
Oh, and the time limit is me being too lazy to add checks if the target server supports `HTTP_RANGE`. I just kill everything if nothing happens after 15 seconds, bit of a gamble.

Comment (hempalex) [inline src/Stnvh/Partial/Zip.php:69]: ob_start suppresses all output from my script
i think it must be done by selectively ignoring errors with "@" or just checking existence before using array index.

set_time_limit prevents to make a long-running import scripts, for example i have a 2GB zip with thousands images on remote server and i need to get only several hundreds of it. But it't time consuming.
I think it must been better way to check http range support - for example - compare request & response range headers.

Comment (stnvh) [inline src/Stnvh/Partial/Zip.php:69]: Will merge PHP 5.3 fixes, and manually refactor error handling soon, including checks for `Accept-Ranges: bytes`

**Comment by stnvh** [comment]
Merged php 5.3 changes into [dev](https://github.com/stnvh/php-partialzip/commit/be449ecc5143ec264b3a542b2787947e292fc859) for staging, and noted that error/output handling needs refactoring (will work on this very soon!)

**Pull request closed without merging**